src/build.rs

```diff
@@ -86,7 +85,7 @@ impl BuildSystem {
     /// Creates a "staging area" (dist/.stage) for storing intermediate build results.
     async fn prepare_staging_dist(&self) -> Result<()> {
         // Prepare staging area in which we will assemble the latest build
-        let staging_dist: &Path = self.cfg.staging_dist.as_path().into();
+        let staging_dist: &Path = self.cfg.staging_dist.as_path();
```

Comment:
If we're doing `as_path()`, we don't need to explicitly mention the type of `staging_dist` now, do we? Or is `as_path()` a generic function?